engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubDeleteBuilder.java

```diff
@@ -25,11 +26,22 @@
 import org.apache.metamodel.query.FilterItem;
 import org.apache.metamodel.schema.Table;
 
-public class DataHubDeleteBuilder extends AbstractRowDeletionBuilder{
+public class DataHubDeleteBuilder extends AbstractRowDeletionBuilder {
+    private static final String GR_ID_COLUMN_NAME = "gr_id";
+    private static final String SOURCE_ID_COLUMN_NAME = "source_id";
+    private static final String SOURCE_NAME_COLUMN_NAME = "source_name";
+
+    enum RecordType {
```

Comment:
Can't we avoid having this somehow? It's one thing that it should eventually go away in DataHub itself, but now it's also leaking to the client. Maybe we can make it so that it's transmitted to the client as a string via some of the services and thus it becomes a dynamic parameter that can eventually also be made dynamic on the server?